test/e2e/scheduling/priorities.go

```diff
@@ -281,14 +279,22 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		Expect(found).To(Equal(true))
 		nodeOriginalMemoryVal := nodeOriginalMemory.Value()
 		nodeOriginalCPUVal := nodeOriginalCPU.MilliValue()
-		err := updateNodeAllocatable(cs, nodeName, int64(10000), int64(12000))
+		err := updateNodeAllocatable(cs, nodeName, int64(10737418240), int64(12000))
```

Comment:
Any reason to set this particular value? Isn't `10000` (greater than incoming limit `5000m`) good enough?

Reply:
This value is for memory not cpu

Reply:
sry, I misread... Let me rephrase: so the original mem val `10000` (greater than incoming limit `3000Mi`) should be put as a form `10737418240`, to represent 10Gi, right?

Reply:
Yes, that is right and that code path was not getting exercised as patch had not effect.